zerver/openapi/zulip.yaml

```diff
@@ -11552,6 +11552,14 @@ paths:
                     - 4
                     - 5
                   example: 1
+                jump_to_sent_message_conversation:
+                  description: |
+                    Whether the user's view should automatically jump to the
+                    conversation where they sent a message.
```

Comment:
Again, this description should specify "... in the web and desktop apps."  at the end, or maybe just start with "Web/desktop app setting for whether ... "; that might be clearer to lead with which apps it impacts.

Reply:
Makes sense. Done.